### PR TITLE
[new release] logs-syslog (0.3.2)

### PR DESCRIPTION
**Pull request by hannesm**
Logs reporter to syslog (UDP/TCP/TLS)

- Project page: <a href="https://github.com/hannesm/logs-syslog">https://github.com/hannesm/logs-syslog</a>
- Documentation: <a href="https://hannesm.github.io/logs-syslog/doc">https://hannesm.github.io/logs-syslog/doc</a>

##### CHANGES:

- update to tls 0.16.0 package split (tls.lwt -> tls-lwt)

### CODE REVIEW

**Comment by hannesm** [comment]
lower bounds failures: ptime https://github.com/ocaml/opam-repository/pull/23347 syslog-message https://github.com/ocaml/opam-repository/pull/23347/commits/dc913b47e862176bd40f9cff05dbfa04a5893b47 (part of #23347)

**Comment by kit-ty-kate** [comment]
The real failure with syslog-message will be fixed by https://github.com/ocaml/opam-repository/pull/23373

File: packages/logs-syslog/logs-syslog.0.3.2/opam [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+homepage: "https://github.com/hannesm/logs-syslog"
+doc: "https://hannesm.github.io/logs-syslog/doc"
+dev-repo: "git+https://github.com/hannesm/logs-syslog.git"
+bug-reports: "https://github.com/hannesm/logs-syslog/issues"
+license: "ISC"
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "1.1.0"}
+  "logs"
```
Comment (kit-ty-kate) [inline packages/logs-syslog/logs-syslog.0.3.2/opam:13]:
```suggestion
  "logs" {>= "0.5.0"}
```
```
#=== ERROR while compiling logs-syslog.0.3.2 ==================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.03.0 | pinned(https://github.com/hannesm/logs-syslog/releases/download/v0.3.2/logs-syslog-0.3.2.tbz)
# path                 ~/.opam/4.03/.opam-switch/build/logs-syslog.0.3.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p logs-syslog -j 71
# exit-code            1
# env-file             ~/.opam/log/logs-syslog-7-2f2e44.env
# output-file          ~/.opam/log/logs-syslog-7-2f2e44.out
### output ###
#       ocamlc src/.logs_syslog_unix.objs/logs_syslog_unix.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.03/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.logs_syslog_unix.objs -I /home/opam/.opam/4.03/lib/astring -I /home/opam/.opam/4.03/lib/bytes -I /home/opam/.opam/4.03/lib/logs -I /home/opam/.opam/4.03/lib/ptime -I /home/opam/.opam/4.03/lib/ptime/os -I /home/opam/.opam/4.03/lib/result -I /home/opam/.opam/4.03/lib/rresult -I /home/opam/.opam/4.03/lib/syslog-message -I src/.logs_syslog.objs -intf-suffix .ml -no-alias-deps -o src/.logs_syslog_unix.objs/logs_syslog_unix.cmo -c -impl src/logs_syslog_unix.ml)
# File "src/logs_syslog_unix.ml", line 25, characters 4-15:
# Error: This expression has type
#          Logs.src ->
#          Logs.level ->
#          over:(unit -> 'a) ->
#          (unit -> 'b) ->
#          ((?header:string ->
#            ?tags:Logs.Tag.set ->
#            ('c, Format.formatter, unit, 'b) format4 -> 'c) ->
#           'd) ->
#          'd
#        but an expression was expected of type
#          Logs.src ->
#          Logs.level ->
#          (unit -> 'e) ->
#          ('f, Format.formatter, unit, 'e) format4 ->
#          ((?header:string -> ?tags:Logs.Tag.set -> 'f) -> 'e) -> 'e
#        Type
#          over:(unit -> 'a) ->
#          (unit -> 'b) ->
#          ((?header:string ->
#            ?tags:Logs.Tag.set ->
#            ('c, Format.formatter, unit, 'b) format4 -> 'c) ->
#           'd) ->
#          'd
#        is not compatible with type
#          (unit -> 'e) ->
#          ('f, Format.formatter, unit, 'e) format4 ->
#          ((?header:string -> ?tags:Logs.Tag.set -> 'f) -> 'e) -> 'e 
#     ocamlopt src/.logs_syslog_unix.objs/logs_syslog_unix.{cmx,o} (exit 2)
```

**Comment by hannesm** [comment]
Thanks for your fix @kit-ty-kate, I pushed it upstream

**Comment by dinosaure** [comment]
Thanks!

**Pull request merged**